src/test/scala/com/softwaremill/realworld/EndpointsSpec.scala

```diff
@@ -1,16 +1,14 @@
-package com.softwaremill
+package com.softwaremill.realworld
```

Comment:
AFAIU it tests only the hello endpoint and could be IMHO removed together with it ;)